roles-responsibilities.md

```diff
@@ -0,0 +1,47 @@
+## Library Carpentry Advisory Committee Roles & Responsibilities
+
+### Library Carpentry Advisory Committee (LC-AC)
+<i>Our Goal:</i>
+The Library Carpentry Advisory Committee is an international network of people involved in the Library Carpentry community. Together we leverage our local networks and advise on ways to foster greater participation in the Library Carpentry community.
+
+<i>Our Approach:</i>
+The Advisory Group discusses, plans, and pursues strategies for involving new and current members in Library Carpentry. 
+
+<i>Our Structure:</i>
+We meet 4 times a year, and members of the Group volunteer to serve three year terms, with increasing levels of responsibility and leadership each year.
```

Comment:
Line 37 says "Serve a 2 year term, with option to continue to 3rd year" so perhaps this could change to "serve two to three year terms."

Reply:
Also in line 11, we meet every two months, so that's 6 times a year, not 4.

Reply:
Line 11: The LC Advisory Group has been meeting every two months, or up to 6 times a year.

Reply:
I think, if https://github.com/LibraryCarpentry/governance/issues/34#issuecomment-1026406516 is approved, the group will switch to meeting quarterly.

Reply:
Oh yes of course, thanks @tobyhodges